core/src/main/java/hudson/model/Queue.java

```diff
@@ -866,7 +841,7 @@ private WaitingItem peek() {
      * @since 1.483
      */
     public List<Item> getApproximateItemsQuickly() {
-        return itemsView.get();
+        return getItemsAsList();
```

Comment:
but you have a smaller diff which is easier to backport... and `Array.asList` is only a single object reference whereas the builder will have a backing equivalent list and the same level of copying. Plus YAGNI favours `Array.asList`